sisl/grid.py

```diff
@@ -139,7 +139,30 @@ def fill(self, val):
         """
         self.grid.fill(val)
 
-    def interp(self, shape, method='linear', **kwargs):
+    def interp(self, shape, order=1, **kwargs):
```

Comment:
Add `mode` to args, say you have a grid where you don't care about periodicity, you may want to do two interps with 
one wrap, another constant.